src/main.rs

```diff
@@ -733,6 +742,7 @@ mod unit_tests {
             #[cfg(feature = "gdb")]
             gdb: false,
             platform: None,
+            vtpm_socket: None,
```

Comment:
I think there should be a new `VtpmConfig` with a single socket member.

Reply:
Agreed and I would go further saying the structure should be called `TpmConfig`.

Reply:
fixed